sys/contrib/ipfilter/netinet/fil.c

```diff
@@ -8009,7 +8024,6 @@ ipf_getnextrule(softc, t, ptr)
 		next = &zero;
 		t->ipt_data = NULL;
 	}
-	it.iri_rule = predict ? next : NULL;
```

Comment:
Why was this line deleted?

Reply:
It's dead code that isn't used, and it's assigning a kernel pointer (vaddr) to a user pointer (capability) so fails to compile.